specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json

```diff
@@ -179,6 +179,46 @@
           }
         }
       }
+    },
+    "/analyze-conversations/jobs/{jobId}:cancel": {
```

Comment:
@tg-msft @johanste I don't see guidelines for exactly how to cancel an LRO. Does this follow the pattern we want? It's similar to what I've seen before, but not entirely clear if this is the right pattern.

Also, and to @bidisha-c as well, why would `:cancel` return an Operation-Location header? Presuming canceling takes a while (plausible), is this a new LRO endpoint apart from the original job that was started, or is it the same and the customer can use either to wait for the "cancelled" status?

Reply:
Yes, that's the typical pattern.  I don't think it's explicitly written down in our guidelines today which we should fix in the new updates.